fbgemm_gpu/test/uvm_test.py

```diff
@@ -78,8 +81,12 @@ def test_uvm_to_cpu(self, sizes: List[int], vanilla: bool) -> None:
     @unittest.skipIf(*gpu_unavailable)
     def test_enum(self) -> None:
         # pyre-ignore[16]
-        assert cudaMemoryAdvise.cudaMemAdviseSetAccessedBy.value == 5
+        if torch.version.hip:
+            assert hipMemoryAdvise.hipMemAdviseSetAccessedBy.value == 5
+        else:
+            assert cudaMemoryAdvise.cudaMemAdviseSetAccessedBy.value == 5
 
+    @skipIfRocm
```

Comment:
Why are all these unit tests skipped?

Reply:
It complains "out of memory" on some of the ROCm devices. We're investigating it.

Reply:
Issue is because a large amount of memory (>1TB) was allocated in the tests. We are checking if this is valid.

Reply:
Is there any progress here?

Reply:
Sorry, not yet. I've submitted a JIRA ticket internally, but it may take some time.

Reply:
I got it. Thanks for your updates.